### PR TITLE
VMO-2943 - seems to fix the jquery race condition

### PR DESCRIPTION
**Pull request by jcbashdown**
The first commit here is probably not ideal but it seems to fix the issue here:

https://github.com/FLOIP/flow-builder/issues/22

Let me know if you have another suggestion though Brett!

Probably more relevant for Safydy, in the second commit, I've taken the opportunity to fix stories which were broken in the package work. I've also moved them out of src - partly as I don't think they should be there and partly to prevent TS errors there breaking the dist build.

To avoid us breaking stories in future, I'm hoping this will keep us honest long term:

https://viamoinc.atlassian.net/browse/VMO-2631 

The last commit is a rebuild of dist. I don't think it's needed to fix that issue but we should do it for the new stuff we've merged to master recently. Once we're happy with this PR I will bump the version, publish and merge.

### CODE REVIEW

**Comment by safydy-r** [comment]
@jcbashdown, any idea why the Toolbar stories was broken ? I tried to find but I have no idea so far
Screen Shot 2020-12-09 at 10 47 04 AM

**Comment by jcbashdown** [comment]
@safydy-r I think that should now be fixed

**Comment by safydy-r** [comment]
Thanks Jacob for the fix, approving

**Review by safydy-r — APPROVED** [review]

File: src/main.ts [hunk 1]
```diff
@@ -3,23 +3,34 @@ import Vuex from 'vuex'
...
+async function initializeBootstrapFramework() {
+  import('bootstrap/dist/css/bootstrap.css')
+  import('bootstrap/dist/css/bootstrap-theme.css')
+
+  const {default: ImportedJquery} = await import('jquery')
+  exportAsGlobal({
+    $: ImportedJquery,
+    jQuery: ImportedJquery})
+
+  await import('bootstrap')
```
Comment (bzabos) [inline src/main.ts:23]: Hi @jcbashdown, I did some testing using DevTools' throttling and was still able to reproduce. I've opted to push these guts into an async function to handle the dependencies more explicitly.

File: src/main.ts [hunk 2]
```diff
@@ -3,23 +3,34 @@ import Vuex from 'vuex'
...
-    .then((jQuery) => {
-      import('bootstrap')
-    })
+function exportAsGlobal(exports) {
+  Object.assign(window, exports)
+}
+
+async function initializeBootstrapFramework() {
+  import('bootstrap/dist/css/bootstrap.css')
+  import('bootstrap/dist/css/bootstrap-theme.css')
```
Comment (bzabos) [inline src/main.ts:16]: I've pulled these imports down into this function as well so our imminent bootstrap-4.5 migration has everything laid out.

**Review by bzabos — APPROVED** [review]
I didn't push up another build commit, because I didn't know what you'd meant by build 😹 . My assumption was that for now "build" implies `build-bundle` producing a lib consumable, is that aligned with your thoughts as well?

Approving. Let me know whether or not you'd like to hash out my changes before merging.

**Comment by smn** [comment]
I'm able to run this now! I am getting a build error (which may be due to me having missed some steps) but the UI is loading. Thank you! 

Screenshot 2020-12-11 at 10 31 42

**Comment by jcbashdown** [comment]
Thanks @bzabos! That looks much better. Thanks for pointing that out also @smn . I'm also getting that but didn't notice it before. Investigating now.

**Comment by jcbashdown** [comment]
@smn Looks like we needed to add a loader declaration for css. Not sure why it wasn't necessary before. Please try again now.

@bzabos - you're right - build-bundle is the first step. Then we need to bump the version in package.json and publish to npm for use in voto5. When we're happy with this I'll go ahead with that. We still have the ticket outstanding to come up with our proper release process but this is good enough for now for the integration work we're focusing on.

**Comment by smn** [comment]
@jcbashdown works for me now! Thank you!

**Comment by jcbashdown** [comment]
Great, thanks for your patience on this Simon. Bumped the version and published. Merging now.

**Pull request merged**